dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -227,7 +227,7 @@ private SQLException classNotFound(Throwable cause) {
      * @return the data source
      * @throws SQLException if an error occurs
      */
-    private <T extends CommonDataSource> T create(final String className, final Hashtable<?, ?> props) throws SQLException {
```

Comment:
This should not have changed from CommonDataSource to Object.
This method is very specific to data sources and should not be used to load a Driver impl.  Driver implementations are not populated with JavaBean properties via reflection, which is what this method does.  In the case where you are given a Driver class name (which is overshooting the scope of this delivery), it should be trivial to either use the ServiceLoader with the given class loader to load java.sql.Driver classes and pick the one that matches the desired class name, or otherwise directly load the Driver impl class & instantiate via reflection.

Reply:
Correct, I initially updated the create method to also be used for drivers but decided against it when refactoring with your changes.  I apparently forgot to update the method signature when I made that change.